packages/react-components/react-image/src/components/Image/useImage.ts

```diff
@@ -6,7 +6,7 @@ import type { ImageProps, ImageState } from './Image.types';
  * Given user props, returns state and render function for an Image.
  */
 export const useImage_unstable = (props: ImageProps, ref: React.Ref<HTMLImageElement>): ImageState => {
-  const { bordered, fit, block, shape = 'square', shadow } = props;
+  const { bordered = false, fit = 'none', block = false, shape = 'square', shadow = false } = props;
```

Comment:
I believe setting `fit = 'none'` here actually changes the styling behavior of this component. Previously this would have defaulted to `undefined` and none of the fit styles would have been applied. Now the `none` fit styles are the default.

Reply:
Fixed!